CONTRIBUTING.md

```diff
@@ -193,11 +193,11 @@ As languages differ quite a lot, here is a set of captures available to you when
 @comment               ; line and block comments
 @comment.documentation ; comments documenting code
 
-@comment.error         ; error-type comments (e.g., `DEPRECATED:`)
-@comment.warning       ; warning-type comments (e.g., `WARNING:`, `FIX:`)
-@comment.hint          ; note-type comments (e.g., `NOTE:`)
+@comment.error         ; error-type comments (e.g. `DEPRECATED:`)
+@comment.warning       ; warning-type comments (e.g. `WARNING:`, `FIX:`)
+@comment.hint          ; note-type comments (e.g. `NOTE:`)
 @comment.info          ; info-type comments
-@comment.todo          ; todo-type comments (e.g-, `TODO:`, `WIP:`)
+@comment.todo          ; todo-type comments (e.g. `TODO:`, `WIP:`)
```

Comment:
Let's not do unrelated changes, shall we. Or at least put it in a different commit

Reply:
Actually you are clason so feel free :smile: I thought it was others doing the change.

Reply:
Yeah, that was a regression that somehow slipped in. I'm fine with pulling this into a separate commit.

Reply:
I think there's a few doc improvements here; I'll put this in a separate PR.